packages/widgetbook/lib/src/addons/zoom_addon/zoom_addon.dart

```diff
@@ -13,13 +13,19 @@ class ZoomAddon extends WidgetbookAddon<double> {
   /// Creates a new instance of [ZoomAddon].
   ZoomAddon({
     this.initialZoom = 1.0,
+    this.precision = 2,
   }) : super(
          name: 'Zoom',
        );
 
   /// Initial zoom level to display when the addon loads.
   final double initialZoom;
 
+  /// The number of decimal places to display and return.
+  ///
+  /// Defaults to 2 for finer control over zoom levels.
+  final int? precision;
+
```

Comment:
We shouldn't expose this property for `ZoomAddon`, since the step is always `0.1`, so it will never be 3 digit precision.